### PR TITLE
Add getActiveAttrib

### PR DESCRIPTION
**Pull request by dzbarsky**

### CODE REVIEW

**Comment by dzbarsky** [comment]
@emilio r?

**Comment by emilio** [comment]
Looks good to me, `r=me` once the gleam PR lands.

File: src/webgl.rs [hunk 1]
```diff
@@ -361,6 +365,18 @@ impl WebGLCommand {
         assert!(error == gl::NO_ERROR, "Unexpected WebGL error: 0x{:x} ({})", error, error);
     }
 
+    fn active_attrib(program_id: u32,
+                     index: u32,
+                     chan: IpcSender<WebGLResult<(i32, u32, String)>>) {
+        let result = if index >= gl::get_program_iv(program_id, gl::ACTIVE_ATTRIBUTES) as u32{
```
Comment (emilio) [inline src/webgl.rs]: nit: space after `u32`

**Comment by emilio** [comment]
@bors-servo: delegate+

Feel free to land this once gleam is published with that nit addressed :)

**Comment by emilio** [comment]
(gleam has just been published)

**Pull request merged**